packages/react/.changeset/strong-turkeys-worry.md

```diff
@@ -0,0 +1,6 @@
+---
+"@gadgetinc/react": patch
+---
+
+- Added an error message to AutoForm when passing in `include/exclude` options alongside child components
```

Comment:
We're throwing a hard error, so maybe it should be "Throw an error" instead.

Reply:
Updated to say that a new error is thrown